### DISCUSSION TITLE
Position Marker using lat long in blank viewer

### DISCUSSION THREAD

**Discussion by Rahul-0108** [iTwin/itwinjs-core#4201] [Q&A] [answered]
Hi , using a blank connection I am trying to position marker using lat long values, but its appearing not in correct postion, it is slightly misplace .This is the code

![image](https://user-images.githubusercontent.com/53996840/187746601-9edc41a3-13f5-4601-925e-48e4a129ee69.png)

Correct position that should be :

![image](https://user-images.githubusercontent.com/53996840/187746851-ae3d5ead-de4e-4fe7-8974-df346730bc2b.png)

but in itwinviewer, its position
![image](https://user-images.githubusercontent.com/53996840/187747095-737ddc10-eb29-4d06-a5dc-da5677faa5ed.png)

marker drawing code
![image](https://user-images.githubusercontent.com/53996840/187747361-fcac02b0-8288-41c4-8cee-948a63ce2a3c.png)

blankconncetionprops
![image](https://user-images.githubusercontent.com/53996840/187747566-992f176e-ea7b-42b7-87ae-71fde6b9eae6.png)

@pmconne @aruniverse

**Comment by aruniverse** [comment]
Please create a sandbox with the above code and share that

**Comment by Rahul-0108** [comment]
@aruniverse 

https://www.itwinjs.org/sandboxes/MyPOC/Dulciana-the-lime-angelfish-246

Please help, I need this asap

also zoom in is not working, so I have used svg to identify the marker in map using 
![image](https://user-images.githubusercontent.com/53996840/187754163-8e1eb91b-efe2-4495-b498-b6bdb362b7ef.png)

**Reply by kabentley** [reply]
your view is centered around Alexandria and your marker is in Baltimore, 40 miles away. Without a projection system (blank connections don't have one) over that distance the curvature of the earth is enough to matter. Move the location of your connection closer to Baltimore and it should work. I also don't understand why you have it at 1000 meters above sea level or why your extents are so large.

**Reply by Rahul-0108** [reply]
@kabentley 
I have a number of points to display as marker all the state , that's why the extent is so large, 
I put z coordinate to 0 in blank Connection prop, still doesn't work
Also the marker is floating while rotating from the top right cube , can anyone suggest anything ??

**Reply by kabentley** [reply]
what are the coordinates of your extents supposed to mean? They are a volume around the location. Z should be -1000, 1000, or something, i would guess

**Comment by bbastings** [comment, marked as answer]
> Also the marker is floating while rotating from the top right cube , can anyone suggest anything ??

If I understand what you mean by floating, as Keith explained you have a point on the surface of the earth/ellipsoid...it's not in the map plane as you don't have a projection system. which is why it was suggested to move your location to minimize the effect of the curvature of the earth.

**Reply by Rahul-0108** [reply]
Thnaks, , looks like what is causing this issue is this only
